src/java.base/share/classes/module-info.java

```diff
@@ -179,6 +179,7 @@
     exports jdk.internal.classfile.components to
         jdk.jfr;
     exports jdk.internal.foreign to
+        java.desktop,
```

Comment:
It's rather frustrating that there's no way to achieve this via a public API. =(